src/Controls/src/Core/Handlers/Items/ItemsViewHandler.Windows.cs

```diff
@@ -32,6 +32,10 @@ public abstract partial class ItemsViewHandler<TItemsView> : ViewHandler<TItemsV
 		bool _emptyViewDisplayed;
 		double _previousHorizontalOffset;
 		double _previousVerticalOffset;
+		double _previousItemSpacing;
+		double _previousHorizontalItemSpacing;
+		double _previousVerticalItemSpacing;
```

Comment:
Are all those set at the same time, at some point it might be a good idea to create a private structure to store this and make the code cleaner.